.pre-commit-config.yaml

```
@@ -11,7 +11,7 @@
 repos:
   # Autoformat: markdown, yaml
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

```suggestion
    rev: v3.1.0
```

The 4 aloha is causing some issues when working on files not changed